jobs/looker-utils/looker_utils/unused_dashboards.py

```diff
@@ -103,12 +103,8 @@ def looker_one_dashboard_download(submission_date, access_token, dashboard_id):
             }        
 
 def main():
```

Comment:
The click parameters need to appear above `def main()`, like here: https://github.com/mozilla/docker-etl/blob/88c68ebd017b742fbc1be341b10e0843cfa570fe/jobs/looker-utils/looker_utils/main.py#L103-L115

Also the `date`, `client_id`, `client_secret` parameters need to be parameters of `main()`

Reply:
oh, got it! Thank you